share/revng/test/configuration/revng/test-abi.yml

```diff
@@ -31,6 +36,11 @@ commands:
       - [i386-regparm-3-systemv-abi]
       - [i386-regparm-2-systemv-abi]
       - [i386-regparm-1-systemv-abi]
+      - [i386-cdecl-microsoft-abi]
+      - [i386-fastcall-microsoft-abi]
+      - [i386-stdcall-microsoft-abi]
+      - [i386-thiscall-microsoft-abi]
+      - [i386-vectorcall-microsoft-abi]
       - [arm-aapcs-abi]
```

Comment:
`repeat-for` is for definition of `sources` only. Drop it.
Same above.

We'd need a YAML schema.

Reply:
What do you mean? I always assumed this was the exact use case `repeat-for` was introduced for 🤔.

Reply:
No, it's just for `sources`.
`repeat-for` does exactly nothing here (and you don't need it).

Reply:
I'll try, if it does indeed changes nothing, then my bad on misunderstanding how it works.

Reply:
Dropped.